### PR TITLE
More global config ref cleanup

### PR DESCRIPTION
**Pull request by ccbrown**
#### Summary
Eliminates most of the remaining utils.Cfg references. I'm so close!

All of this is pretty straight-forward. Nothing too interesting except maybe the change to the command interface.

#### Ticket Link
N/A

#### Checklist
N/A

### CODE REVIEW

File: app/command_loadtest.go [hunk 1]
```diff
@@ -66,16 +66,17 @@ type LoadTestProvider struct {
 }
 
 func init() {
-	if !utils.Cfg.ServiceSettings.EnableTesting {
-		RegisterCommandProvider(&LoadTestProvider{})
-	}
```
Comment (ccbrown) [inline app/command_loadtest.go]: 1. The configuration is not loaded when `init` is called.
2. The condition is wrong. We want the command to be present when `EnableTesting` is `true`.

Comment (hmhealey) [reply app/command_loadtest.go]: That doesn't sound good. Did that allow the testing command to be ran when it shouldn't have been enabled?

Comment (ccbrown) [reply app/command_loadtest.go]: You were able to run it on any server, but on servers without `EnableTesting`, it just wouldn't do anything. Try it on pre-release. No errors like you would get with a command that doesn't exist. But no response either.

**Comment by coreyhulen** [comment]
@ccbrown feel free to merge

**Pull request merged**